gradle/build-hadoop-distros.gradle

```diff
@@ -28,6 +28,7 @@ project('spring-xd-hadoop:hadoop24') {
 		runtime ("org.apache.hadoop:hadoop-mapreduce-client-jobclient:$hadoop24Version")
 		runtime ("org.apache.hadoop:hadoop-streaming:$hadoop24Version")
 		runtime ("org.apache.hadoop:hadoop-yarn-common:$hadoop24Version")
+		configurations.runtime.exclude(group: 'org.xerial.snappy')
```

Comment:
I added the specific modules to excludes to fix spring-xd-dirt and spring-xd-test poms which are critical to spring-xd-module-parent  (also spring-xd-module and spring-xd-module-spi). As long as we can still run SingleNodeApplication from an external maven project. Unfortunately there are no automated tests for this.